i18n/zh/docusaurus-plugin-content-docs/current/developer/mappers/index.md

```diff
@@ -3,24 +3,136 @@ title: Mappers
 id: mappers-index
 ---
 
-## Overview
-Mapper is an application that is used to connect and control devices. The responsibilities of mapper are as follows:
+## 概述
+Mapper是用来连接并控制设备的应用，它负责的功能如下：
 
-- Scan and connect to the device.
-- Report the actual state of twin-attributes of device.
-- Map the expected state of device-twin to actual state of device-twin.
-- Collect telemetry data from device.
-- Convert readings from device to format accepted by KubeEdge.
-- Schedule actions on the device.
-- Check health of the device.
+- 扫描并连接设备.
```

Comment:
End with "。" punctuation

Reply:
All Done in #439. Since there was a problem with my account before, all PR was hidden, so I resubmitted a PR. Now that my account has returned to normal, this PR will be closed. Please see #439  for the latest, thanks @WillardHu